src/components/partials/IdentitySearch/CustomNodeModal.vue

```diff
@@ -115,6 +124,10 @@ export default class CustomNodeModal extends Vue {
             set<ChainInfo>(StoreKey.CustomNode, customNode);
             this.$emit("save", customNode);
         } catch (e) {
+            if (e.message === "InvalidAddressError") {
+                return (this.error =
+                    "The address is invalid. It should start with 'wss://'.");
+            }
```

Comment:
@sosan-neikbeen a bit late for an early exit :D Nice, but use a standard if/else statement here.